subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/MavenBuildExperimentRunner.java

```diff
@@ -97,28 +99,30 @@ public void doRun(BuildExperimentSpec experiment, MeasuredOperationList results)
 
     private InvocationSettings createInvocationSettings(MavenBuildExperimentSpec experimentSpec) {
         File outputDir = getFlameGraphGenerator().getJfrOutputDirectory(experimentSpec);
-        return new InvocationSettings(
-            experimentSpec.getInvocation().getWorkingDirectory(),
-            getProfiler(),
-            true,
-            outputDir,
-            new BuildInvoker() {
-                @Override
-                public String toString() {
-                    return "Maven";
-                }
-            },
-            false,
-            null,
-            ImmutableList.of(experimentSpec.getInvocation().getMavenVersion()),
-            experimentSpec.getInvocation().getTasksToRun(),
-            ImmutableMap.of(),
-            null,
-            warmupsForExperiment(experimentSpec),
-            invocationsForExperiment(experimentSpec),
-            false,
-            ImmutableList.of(),
-            CsvGenerator.Format.LONG);
+        return new InvocationSettings.InvocationSettingsBuilder()
+            .setProjectDir(experimentSpec.getInvocation().getWorkingDirectory())
+            .setProfiler(getProfiler())
+            .setBenchmark(true)
+            .setOutputDir(outputDir)
+            .setInvoker(
+                new BuildInvoker() {
+                    @Override
+                    public String toString() {
+                        return "Maven";
+                    }
+                })
+            .setDryRun(false)
+            .setScenarioFile(null)
```

Comment:
Can we remove these calls now, given that we are not setting anything?

Reply:
Yep